pkg/controller/plan/adapter/vsphere/validator_test.go

```diff
@@ -0,0 +1,121 @@
+//nolint:errcheck
+package vsphere
+
+import (
+	v1beta1 "github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
+	"github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1/plan"
+	"github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1/ref"
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/model/vsphere"
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/web"
+	model "github.com/konveyor/forklift-controller/pkg/controller/provider/web/vsphere"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Mock inventory struct and methods for testing
+type mockInventory struct{}
+
+func (m *mockInventory) Find(resource interface{}, ref ref.Ref) error {
+	switch res := resource.(type) {
+	case *model.Workload:
+		*res = model.Workload{
+			VM: model.VM{
+				NICs: []vsphere.NIC{
+					{MAC: "mac1"},
+					{MAC: "mac2"},
+				},
+				GuestNetworks: []vsphere.GuestNetwork{
+					{MAC: "mac1"},
+				}},
+		}
+		if ref.Name == "full_guest_network" {
+			res.VM.GuestNetworks = append(res.VM.GuestNetworks, vsphere.GuestNetwork{MAC: "mac2"})
+		}
+	}
+	return nil
+}
+
+func (m *mockInventory) Finder() web.Finder {
+	return nil
+}
+
+func (m *mockInventory) Get(resource interface{}, id string) error {
+	return nil
+}
+
+//nolint:nilnil
```

Comment:
I think you can just put it at the top of the file

Reply:
looks like i don't :)